.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+<!-- Thank you for contributing! -->
+
+## Description
+
+<!-- Please insert your description here and provide especially info about the "what" this PR is solving -->
+
+### Additional context
+
+<!-- e.g. is there anything you'd like reviewers to focus on? -->
+
+### Issue reference
+
+<!-- Please insert a link to the solved issue. If none, create one for this PR and then reference it here -->
```

Comment:
Do we want to add links to our internal Jira? Or just add the IDs of the issues? I don't mind anything, I just try to not share links to internal software in public :)

Reply:
I know, but this is happening right now. So I want to make this clear and discuss this if we do this or not.

Reply:
We are allowed to do this. So I am letting this here.